src/ert/ensemble_evaluator/evaluator.py

```diff
@@ -82,14 +65,20 @@ def __init__(self, ensemble: Ensemble, config: EvaluatorServerConfig):
         self._max_batch_size: int = 500
         self._batching_interval: float = 2.0
         self._complete_batch: asyncio.Event = asyncio.Event()
+        self._server_started: asyncio.Event = asyncio.Event()
+        self._clients_connected: set[bytes] = set()
```

Comment:
I think `self._connected_clients` would be a better name. `clients_connected` sounds more like a boolean

Reply:
Also, that is client? Is it the Client we use for sending/receiving, or is it referring to the monitor for UI?

Reply:
Maybe we can rename it to monitors

Reply:
Hm. I was trying to preserve the naming conventions we have before (which was clients), but maybe having it explicit as `Monitor` would make more sense. Then I would change `handle_client` -> `handle_monitor` as well.

Reply:
I would do it in another PR to keep the semantics as it was before.